### PR TITLE
Add features: Project Release

### PR DESCRIPTION
**Pull request by manq2010**
HTML/CSS & JavaScript capstone project - Conference page
---
In this release, the following was done:
* Added 2 pages 
* Each page mobile responsive
* Added interactions to the links and mobile menu
* Feature and main-program sections are dynamically added

[link to loom recording](https://www.loom.com/share/7672b7d2b60b46e08dc9d22f517b4af7)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+        <buttton class="btn fs-4 d-md-none">
+            <span class="navbar-toggler-icon"></span>
+        </buttton>
+        <a class="logo-nav d-none d-md-block text-decoration-none" href="index.html">
+            <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="currentColor" class="bi bi-intersect"
+                viewBox="0 0 16 16">
+                <path
+                    d="M0 2a2 2 0 0 1 2-2h8a2 2 0 0 1 2 2v2h2a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2v-2H2a2 2 0 0 1-2-2V2zm5 10v2a1 1 0 0 0 1 1h8a1 1 0 0 0 1-1V6a1 1 0 0 0-1-1h-2v5a2 2 0 0 1-2 2H5zm6-8V2a1 1 0 0 0-1-1H2a1 1 0 0 0-1 1v8a1 1 0 0 0 1 1h2V6a2 2 0 0 1 2-2h5z" />
+            </svg>
+            Soweto Co
```
Comment (victorokoroji) [inline index.html:44]: - [ ] The project you presented in your video is kind of different from your UI. Your UI is currently not looking like the design template you are given  Please check it out 👍🏽 

Your Design 👇🏽 

image

Design Template 👇🏽 

image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @manq2010, 

Good job so far! 🤝 
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 1 above)
Comment (manq2010) [reply index.html:44]: Thanks for the feedback, this has been fixed.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,92 @@
...
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+- Node.js installed
+
 ### Setup
 
-### Install
+#### Set-up GitHub Actions
```
Comment (christianbiring1) [inline README.md:30]: [OPTIONAL] 
- [ ] Kindly you can consider walking the user through the process of getting the local copy of the project  form GitHub
`hint: You may say: git clone <repos link> `
`cd repos <repos directory_name>`
`open in IDE`
`Go live with live server`
- [ ] You may also consider doing the same for the installation, setup and prerequisites, `Be as much detailed as possible.`

File: js/index.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+const moreSpeaker = document.getElementById('more-speakers');
+
+moreSpeaker.addEventListener('click', () => {
+  speakerCard.forEach((speaker, index) => {
+    if (index >= 2) {
+      moreCardSpeaker += contentSpeakers(speaker);
+    }
+  });
+  speakerContainer.insertAdjacentHTML('afterbegin', moreCardSpeaker);
+
```
Comment (christianbiring1) [inline js/index.js:157]: [OPTIONAL]
- [ ] The `see_more` feature was supposed to be visible only on the mobile version. 
On the desktop, all the speakers are supposed to be visible

**Review by christianbiring1 — APPROVED** [review]
Hi @manq2010 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
APPROVED

## Highlights
- [ ] All linters checks are green
- [ ] Descriptive PR `title` and `summary`
- [ ] Appropriate credential was given to `Cindy` in the README.
## Optional suggestions
- [ ] I hih;y recommend you i,plement them for they will help take your project to the next level
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 3 above)
Comment (manq2010) [reply js/index.js:157]: thanks @christianbiring1 , will work on that

**Pull request merged**